### DISCUSSION TITLE
Isar Inspector Dev

### DISCUSSION THREAD

**Discussion by levyhnunes** [isar/isar#560] [Help] [answered]
Hi, my tech lead asked if Isar Inspector is safe? He fears that project data is being shared, as in version 3.0.0-dev.8 a web link is generated. Is it possible to disable?

**Comment by simc** [comment, marked as answer]
The data never ever leaves your device. Just the UI is hosted online but your browser connects to your local machine using a websocket. No server is involved.

You can verify that in the Chrome inspector. Also try opening the inspector and then disable internet on your Mac. It will continue working.

Furthermore it is impossible to compile inspector support into a release app. It will be tree shaken by the Dart compiler.

If you also want to disable it for debug builds just pass `inspector: false` to `Isar.open()`

**Reply by vleeuwenmenno** [reply]
I was programming in the plane the other day to kill some time of my 14h flight and couldn't see my db so had to write extra API calls just to see my database tables. Is there any way to run this locally. Or a program that can load in the file and show a UI?
